next/components/forms/segments/AccountSections/IntroSection/IntroSection.tsx

```diff
@@ -4,6 +4,7 @@ import MoneyServiceIcon from '@assets/images/account/money-service-icon.svg'
 import PoolServiceIcon from '@assets/images/account/pool-service-icon.svg'
 import useAccount from '@utils/useAccount'
 import AccountSectionHeader from 'components/forms/segments/AccountSectionHeader/AccountSectionHeader'
+import ActualBlock from 'components/forms/segments/AccountSections/IntroSection/ActualBlock'
```

Comment:
perhaps "AnnouncementBlock" is a better name ?

otherwise it sounds more like "Wow, I can't believe this is an _actual_ block!" 🙂

Reply:
Hahaha, AnnouncementBlock seems better